english/chocolateOrSleep/chocolateOrSleep.md

```diff
@@ -0,0 +1,2 @@
+Would you like some more [chocolate](../search_for_chocolate/search_for_chocolate.md) or 
+would you like to [sing](../sing/sing.md)?
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.